tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "emitDeclarationOnly": true,
     "isolatedDeclarations": true,
     "target": "es2017",
-    "lib": ["es2017"],
+    "lib": ["es2017", "DOM"],
```

Comment:
Instead of pulling in DOM types, let's declare `AbortSignal` in files where it's needed, with only the properties/methods it needs.

#### In `parallel.ts`

```ts
type AbortSignal = {
  get aborted(): boolean
}
```

#### In `retry.ts`

```ts
type AbortSignal = {
  throwIfAborted(): void
}
```

Reply:
@SaidbekAbdiganiev Did you resolve this comment on accident? I don't see any commits that address it.

Reply:
Forgot to push to the PR, my bad 😅